examples/variables.tf

```diff
@@ -51,10 +51,25 @@ variable storage_capacity {
 
 variable num_hosts {
   description = "The number of hosts."
-  default     = ""
+  default     = 1
 }
 
 variable provider_type {
   description = "Determines what additional properties are available based on cloud provider. Acceptable values are ZEROCLOUD and AWS with AWS as the default value."
```

Comment:
I thought you are removing the references to ZEROCLOUD

Reply:
Already removed in master branch.